docs/data/material/components/breadcrumbs/CustomizedBreadcrumbs.tsx

```diff
@@ -6,22 +6,27 @@ import HomeIcon from '@mui/icons-material/Home';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
 const StyledBreadcrumb = styled(Chip)(({ theme }) => {
-  const backgroundColor =
-    theme.palette.mode === 'light'
-      ? theme.palette.grey[100]
-      : theme.palette.grey[800];
   return {
-    backgroundColor,
+    backgroundColor: theme.palette.grey[100],
```

Comment:
```suggestion
    backgroundColor: (theme.vars || theme).palette.grey[100],
```

Not that important right now, but maybe we can be consistent, as people may have hard time figuring out why in some places we have theme and why theme.vars. In this case it doesn't matter, this is just for improving the demo, we can do a follow up cleanup in the future.

Reply:
Gotcha.